Week2/nodeapp.js

```diff
@@ -0,0 +1,91 @@
+const mysql = require('mysql');
+const prompt = require('prompt');
+const util = require('util');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+const input = util.promisify(prompt.get.bind(this));
+async function queryQuestions() {
+  try {
+    console.log(`What do you want? Select the number of the question to be answered.
+-----------------
+1-Get the capital city of any country you want. 
+2-List all the languages spoken in the region you want. 
+3-Get the number of cities in which language you want is spoken .
+4-Get the countries in the region you want with the language you want as the official language ?
+5-List all the continents with the number of languages spoken in each continent`);
+    prompt.start();
+    console.log('enter the number:');
+    const result = await input(['userInput']);
+    const userInput = result.userInput;
+    await execQuery(`use world`);
+
+    if (userInput === '1') {
+      console.log('Enter the country name:');
+      const result = await input(['userInput']);
+      const country = result.userInput;
+      const data = await execQuery(
+        `with result as 
+        (select country.name as Country, city.name as Capital
+        from world.country join world.city on country.Capital=city.ID)
+        select Country, Capital from result where Country like '%${country}%'`,
```

Comment:
What would happen if the user enters the following query:
Spain; DROP TABLE country
You should be using prepared statements to guard your app against a hacker attack.
You can find an example here https://github.com/mysqljs/mysql#preparing-queries